cmd/circle/import.go

```diff
@@ -79,41 +79,10 @@ func (c *importCommand) run(*kingpin.ParseContext) error {
 		c.harnessAccount,
 		c.harnessOrg,
 		c.harnessToken,
-		c.githubToken,
-		c.gitlabToken,
-		c.bitbucketToken,
 		c.harnessAddress,
 	)
 	importer.Tracer = tracer_
 
-	// create a scm client to verify the token
-	// and retrieve the user id.
-	client := util.CreateClient(
-		c.githubToken,
-		c.gitlabToken,
-		c.bitbucketToken,
-		c.githubURL,
-		c.gitlabURL,
-		c.bitbucketURL,
-		c.skipVerify,
-	)
-
-	// get the current user id.
-	user, _, err := client.Users.Find(ctx)
-	if err != nil {
-		log.Error("cannot retrieve git user", nil)
-		return err
-	}
-
-	// provide the user id to the importer. the user id
-	// is required by the connector despite the fact that
-	// it can be retrieved using the token itself (like we just did)
-	importer.ScmLogin = user.Login
-
-	log.Debug("verified user and token",
-		slog.String("user", user.Login),
-	)
```

Comment:
I'm ok with removing this from import, but that means it would need to happen in the export and be present in the export json payload